### PR TITLE
Fix for known issue - Purview registration

### PR DESCRIPTION
**Pull request by amanjeetsingh**
Added an additional workaround to address Purview registration issue

**This PR fixes**

### CODE REVIEW

File: docs/EnterpriseScaleAnalytics-KnownIssues.md [hunk 1]
```diff
@@ -73,7 +73,7 @@ This error message appears, if the Purview soft-limit has been reached inside yo
...
+This error message appears, when deploying Purview and the `Microsoft.EventHub`, `Microsoft.Storage` and/or `Microsoft.Purview` Resource Provider is not registered for the subscription or it was registered during early preview phase. If Purview resource provider is already registered, please un-register Purview resource provider and then re-register it. We have released a fix to our Deploy To Azure Buttons to overcome this issue until it gets fixed on the Purview side. If you still run into this problem, please open an Issue in this repo and start (re-)registering the three Resource Providers manually through the `Azure Portal`, `Azure Powershell` or `Azure CLI` as mentioned [here](https://docs.microsoft.com/azure/azure-resource-manager/management/resource-providers-and-types). Once that is done, use the "Redeploy" button on the failed deployment to restart the deployment and successfuly deploy the components.
```
Comment (marvinbuss) [inline docs/EnterpriseScaleAnalytics-KnownIssues.md:76]:
```suggestion
This error message appears, when deploying Purview and the `Microsoft.EventHub`, `Microsoft.Storage` and/or `Microsoft.Purview` Resource Provider (RP) is not registered for the subscription or it was registered during early preview phase. If the `Microsoft.Purview` RP is already registered, please un-register and re-register it. We have released a fix to our Deploy To Azure Buttons to overcome this issue until it gets fixed on the Purview side. If you still run into this problem, please open an Issue in this repo and start (re-)registering the three Resource Providers manually through the `Azure Portal`, `Azure Powershell` or `Azure CLI` as mentioned [here](https://docs.microsoft.com/azure/azure-resource-manager/management/resource-providers-and-types). Once that is done, use the "Redeploy" button on the failed deployment to restart the deployment and successfully deploy the components.
```

**Review by marvinbuss — CHANGES_REQUESTED** [review]
Please see my comments below.

File: docs/EnterpriseScaleAnalytics-KnownIssues.md (hunk 1 above)
Comment (marvinbuss) [reply docs/EnterpriseScaleAnalytics-KnownIssues.md:76]: @amanjeetsingh please see my suggestions above. I tried to make some of the words consistent like RP.

Comment (amanjeetsingh) [reply docs/EnterpriseScaleAnalytics-KnownIssues.md:76]: Thanks @marvinbuss

**Review by marvinbuss — APPROVED** [review]
LGTM!

**Pull request merged**